app/src/main/java/com/learnig/android/cleanarchitecture/core/di/AppModule.kt

```diff
@@ -0,0 +1,21 @@
+package com.learnig.android.cleanarchitecture.core.di
+
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.koin.android.ext.koin.androidApplication
+import org.koin.dsl.module
+
+val appModule = module {
+    factory {
+        LinearLayoutManager(androidApplication())
```

Comment:
Przekazanie contextu przez `androidContext()` będzie w tym przypadku chyba bardziej odpowiednie. Nawet jeśli gdzieś na końcu w obu przypadkach będzie używany ten sam kontekst aplikacji, to przynajmniej patrząc na moduł od razu wiadomo, że jest tutaj potrzebny właśnie kontekst jako zależność.